### ISSUE TITLE
Move blocking_session_id in BlitzWho output beside session_id for easier block identification

### ISSUE THREAD

**Issue by davidhooey** [BrentOzarULTD/SQL-Server-First-Responder-Kit#3159]
It would be much easier to identify blocking chains when the `session_id` and `blocking_session_id` columns are beside each other.

**Comment by erikdarlingdata** [comment]
This is a neat idea! Just make sure to CTRL+F for `blocking_session_id` to catch all the place where column order would need to be changed.

**Comment by davidhooey** [comment]
I can do the change. My only concern is if it will cause issues with the storage table? May have to keep the existing order for table inserts supporting older versions of the table? Or do we just change the table column order as well causing issues with the older version of the schema?

From: Erik Darling ***@***.***>
Date: Wednesday, November 9, 2022 at 1:50 PM
To: BrentOzarULTD/SQL-Server-First-Responder-Kit ***@***.***>
Cc: davidhooey ***@***.***>, Assign ***@***.***>
Subject: Re: [BrentOzarULTD/SQL-Server-First-Responder-Kit] Move blocking_session_id in BlitzWho output beside session_id for easier block identification (Issue #3159)

This is a neat idea! Just make sure to CTRL+F for blocking_session_id to catch all the place where column order would need to be changed.

**Comment by erikdarlingdata** [comment]
Yeah, I wouldn't mess with the table column order, if the goal is purely cosmetic.

**Comment by davidhooey** [comment]
Great! I will start working on a pull request.
________________________________

**Comment by BrentOzar** [comment]
Thanks for the pull request! Looks good. Merging into the dev branch, will be in the December release with credit to you in the release notes.

**Comment by davidhooey** [comment]
@BrentOzar this change did not make it into the 2022-12-13 release for some reason. The `blocking_session_id` is in its original location in the `20221213` tag:

https://github.com/BrentOzarULTD/SQL-Server-First-Responder-Kit/blob/20221213/sp_BlitzWho.sql#L710

However, the change is shown in the `dev` branch:

https://github.com/BrentOzarULTD/SQL-Server-First-Responder-Kit/blob/dev/sp_BlitzWho.sql#L690

Thanks!

**Comment by BrentOzar** [comment]
Sorry about that! Check now.
